homework-classes/RepoView.js

```diff
@@ -18,9 +18,40 @@
      * Renders the repository details.
      * @param {Object} repo A repository object.
      */
+
     render(repo) {
-      // TODO: replace this comment and the console.log with your own code
       console.log('RepoView', repo);
+      const repoContainer = document.querySelector('.repo-container');
+      const select = document.querySelector('.repo-select');
+
+      function addRow(table, title, repoText) {
```

Comment:
When using classes, it is a best practice to use class methods instead of embedded functions. In this case, because the function body does not reference the `this` value, this could be a **static** method:

```js
render(repo) {
  ...
  RepoView.addRow(table, 'Description:', repo.description, 'td');
  ...
}

static addRow(table, title, repoText) {
  ...
}
```

Reply:
I don't understand the static mode but I understand class and write this js page again

Reply:
You can also use a 'regular' method like this:

```js
render(repo) {
  ...
  this.addRow(table, 'Description:', repo.description, 'td');
  ...
}

addRow(table, title, repoText) {
  ...
}
```

Static methods are preferred (but not required) if the method body does not contain a reference to the `this` value, which holds true for your `addRow` method. If you _do_ use a static method then you can't call it with `this.myMethodName()`. Instead, you must the class name, e.g. `MyClass.myMethodName()`.

See also: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static